docs/maps/skylands.md

```diff
@@ -0,0 +1,41 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Skylands
+
+***
+
+#### ![skylandsicon](../assets/maps/skylands/skylands-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** Fortress built on floating islands.
```

Comment:
Please remove "Fortress" and replace this with something else. "Fortress" doesn't describe the map too well. Perhaps try including the fountain and center part of the map in the description? Or maybe something more simple like: "Floating islands cleansed from the world below."?